.travis.yml

```diff
@@ -42,6 +42,7 @@ matrix:
 install:
     - git clone git://github.com/astropy/ci-helpers.git
     - source ci-helpers/travis/setup_conda_$TRAVIS_OS_NAME.sh
+    - pip install pytest==$PYTEST_VERSION
```

Comment:
you don't need this

Reply:
I added this because I wasn't sure why I got this failure: https://travis-ci.org/astropy/pytest-remotedata/jobs/282764253

Reply:
weird, it should be installed: https://travis-ci.org/astropy/pytest-remotedata/jobs/282764253#L780